crates/pessimistic-proof/src/global_index.rs

```diff
@@ -46,8 +46,7 @@ impl From<U256> for GlobalIndex {
 }
 
 impl GlobalIndex {
-    #[allow(unused)]
-    fn to_u256(self) -> U256 {
+    pub fn to_u256(self) -> U256 {
```

Comment:
It wasn't possible to have an `Into` implementation?

Reply:
yeah I moved it to From/Into implementation